### ISSUE TITLE
New Login Ui

### ISSUE THREAD

**Issue by Aditya01237** [Clueless-Community/Spectrum-UI#144]
Can we add new login and registration UI to it.

**Comment by Apreksha** [comment]
Hello. I'm a GSSoC'23 contributor. I want to add new login UI. Could you please assign me this task.

**Comment by ashdude1401** [comment]
@Aditya01237 could you please share more info regarding your Idea and  design or Figma file that you want to implement .

**Comment by Aditya01237** [comment]
We can add module of Login ui and add differnt ui to it.

<img src= "https://github.com/Clueless-Community/Spectrum-UI/assets/80575917/319a1751-2629-4997-b019-8e8b965001b6" alt="splash" width="220" height="450"/> <img src= "https://github.com/Aditya01237/ChatApp/assets/80575917/459a0eea-8482-4af2-9376-5f5062c9c1c6" alt="splash" width="220" height="450"/>

**Comment by ashdude1401** [comment]
Closing Pull Request for @Aditya01237: Login and Sign-In Functionality not in Scope for New Design

Dear @Aditya01237,

Thank you for submitting your Pull Request regarding the login and sign-in functionality. After careful consideration, we have decided not to include these features in the app as they fall outside the primary scope of its functionalities.

We appreciate your effort in raising this issue and your dedication to improving the app. We encourage you to redirect your focus towards identifying other potential bugs or suggesting enhancements that align with the app's core objectives.

Once again, thank you for your contribution and your understanding. We look forward to your continued involvement in making our app even better.

Best regards,
Ashhutosh Jha/Spectrum UI
